test/unit/app-lifecycle.spec.js

```diff
@@ -111,6 +111,10 @@ describe('App-Lifecycle', function() {
       });
     });
 
+    it('should start with _isRestarting flag set to false', function() {
+      expect(this.myApp._isRestarting).to.equal(false);
```

Comment:
Let's test this via `isRestarting()`  If possible we should test from only public API

Reply:
Gotcha, though I did see examples of test referencing `_isRunning`, so that's what made me think this was 👍 

Anyway, I'll make this change to only refer to the public API.